TweetBook/Contract/V1/ApiRoutes.cs

```diff
@@ -28,5 +28,16 @@ public static class Identity
 
             public const string Refresh = Base + "/identity/refresh";
         }
+
+        public static class Tags
+        {
+            public const string GetAll = Base + "/tags";
+
+            public const string Create = Base + "/tags";
+
+            public const string Get = Base + "/tags";
```

Comment:
I see that GetAll, Create and Get are all equal variables, could you eliminate the duplication?